src/app/components/split-view/index.html

```diff
@@ -0,0 +1,239 @@
+<sky-tabbed-demo-page
+  pageTitle="Split view"
+>
+  <sky-demo-page-summary>
+    <p>
+      The split view component allows users to work through a list of items, while displaying each item's details in context. The split view component consists of two children components: the drawer and the workspace.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-example>
+    <sky-split-view-demo></sky-split-view-demo>
+  </sky-demo-page-example>
+
+<sky-tabset id="content-tabs">
+
+  <sky-tab
+    tabHeading="Design">
+
+    <sky-demo-page-content sectionHeading="Usage">
+      <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use a split view if a user needs to work through a list of items with the intent of taking an action on each item in the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/use-when.png" />
+            <div class="caption caption-success sky-text-success">Use a split view when the user will take the same action for each item in the list.</div>
```

Comment:
I'm seeing a narrow column of text above this first image. Is that deliberate?
<img width="713" alt="Screen Shot 2019-07-17 at 2 17 19 PM" src="https://user-images.githubusercontent.com/18354182/61400335-08f3b800-a89e-11e9-811f-cdf4244a7bea.png">

Reply:
This is the intended behavior. I was talking about it with Adam on Tuesday, and we agreed to leave this as is for now. I'll revisit this with him and Todd down the road.